packages/instanceserver/src/NetworkFunctions.ts

```diff
@@ -319,18 +319,11 @@ export async function handleDisconnect(network: SocketWebRTCServerNetwork, peerI
     const app = Engine.instance.api as Application
 
     if (!instanceServerState.isMediaInstance)
-      app.service(messagePath).create(
-        {
-          instanceId: instanceServerState.instance.id,
-          text: `${userName} left`,
-          isNotification: true
-        },
-        {
-          'identity-provider': {
-            userId: userId
-          }
-        } as any
-      )
+      app.service(messagePath).create({
```

Comment:
this is not correct, i fixed this in dev, please revert this. the identity provider is needed